codeforamerica/vita-min
# Fyst 387 Add phone number to NC return header

## Link to pivotal/JIRA issue
- https://codeforamerica.atlassian.net/browse/FYST-387
## Is PM acceptance required? (delete one)
- Yes - don't merge until JIRA issue is accepted!
## What was done?
- Adds the phone number to the return header; pulled from DF per the story
- This is the first state-divergent thing in the return header so would appreciate any input on the approach there
## How to test?
- Added a phone_number writer to df data and the ability to edit from the fed info page so acceptance testers can see whatever phone number they add show up in the xml
- unit specs for df data and return header
- feature spec for editing the df xml
## Screenshots (for visual changes)
<img width="695" alt="image" src="https://github.com/user-attachments/assets/45697153-eed5-46da-878b-2e0539b47a97">
